tests/unit/Core/System/CustomEntity/Schema/SchemaUpdaterTest.php

```diff
@@ -186,6 +187,146 @@ public static function associationPairsProvider(): \Generator
         ];
     }
 
+    /**
+     * @param list<array{name: string, fields: string}> $entities
+     * @param array<string, list<string>> $notExpectedSchema
+     * @param array<string, list<string>> $expectedNonExistTableNames
```

Comment:
```suggestion
     * @param list<string> $expectedNonExistTableNames
```

then the var annotation is probably also not necessary